odoo/tools/translate.py

```diff
@@ -1192,6 +1193,15 @@ def _export_imdinfo(self, model: str, imd_per_id: dict[int, ImdInfo]):
                     _logger.exception("Failed to extract terms from %s %s", xml_name, name)
                     continue
                 for term_en, term_langs in translation_dictionary.items():
+                    term_en_unescaped = html_stdlib.unescape(term_en)
+                    value_en_unescaped = html_stdlib.unescape(value_en)
+                    re_attr_no_export = r'(src|href|data-oe-expression)=[\'"]' + re.escape(term_en_unescaped) + r'[\'"]'
+                    if re.search(re_attr_no_export, value_en_unescaped):
+                        # That's not perfect, we could check that the term
+                        # is ONLY in src/href attributes, but that's not perfect
+                        # either, because we could have this HTML:
+                        # <img src="X"/> <div style="background-image: url(X)"/>
+                        continue
```

Comment:
This seems to work half-way. It mostly works but a few links go through and I don't get how/why.